src/main/java/gov/cabinetoffice/gap/adminbackend/config/JacksonConfig.java

```diff
@@ -22,6 +23,7 @@ public ObjectMapper getObjectMapper() {
         objectMapper.disable(FAIL_ON_UNKNOWN_PROPERTIES);
         objectMapper.registerModule(new JavaTimeModule());
         objectMapper.registerModule(new ParameterNamesModule(JsonCreator.Mode.PROPERTIES));
+        objectMapper.setSerializationInclusion(JsonInclude.Include.NON_NULL);
```

Comment:
This has potential to affect other places which scares me a bit

Reply:
Yeah depends on wether we check if a field exists by name rather than value